src/coloring/coloring_driver.cpp

```
@@ -0,0 +1,136 @@
+#include "coloring/edgeColoring.hpp"
+#include "coloring/sequentialVertexColoring.hpp"
+
+namespace pgrouting {
+namespace drivers {
+
+void do_coloring(
+        const std::string &edges_sql,
+
+        Which which,
+
+        II_t_rt* &return_tuples,
```

Cppcheck (2.20.0)

[style] The function 'set_ids' is never used.

(unusedFunction)